app/templates/grunt/config/eslint.js.ejs

```diff
@@ -1,13 +1,16 @@
 module.exports = {
     options: {
-        envs: ['browser'],
-        configFile: './node_modules/eslint-config-nikita/index.js'
+        envs: ['browser', 'jest'],
+        configFile: './node_modules/eslint-config-nikita/index.js',
+        rules: {
+            // override rules here
+        }
     },
-    check: ['<%%= paths.src %>/js/**/*.js'],
```

Comment:
@axten please ignore main.js here. Otherwise eslint will fail with generated files

Reply:
moved files to  generated/